resources/providers/awslib/iam/get_policies.go

```diff
@@ -0,0 +1,102 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package iam
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	iamsdk "github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/elastic/cloudbeat/resources/fetching"
+	"github.com/elastic/cloudbeat/resources/providers/awslib"
+	"net/url"
+)
+
+func (p Provider) GetPolicies(ctx context.Context) ([]awslib.AwsResource, error) {
+	var policies []awslib.AwsResource
+
+	listPolicies, err := p.client.ListPolicies(ctx, &iamsdk.ListPoliciesInput{OnlyAttached: true})
```

Comment:
Does `ListPolicies` supports pagination? if so I think we should add it.

Reply:
Good catch, thanks!